feat(genapi): add troubleshooting index (#4588)
scaleway/docs-content: pages/generative-apis/troubleshooting/index.mdx

---
meta:
  title: Generative APIs - Troubleshooting
  description: Find troubleshooting resources for resolving issues with Generative APIs.
content:
  paragraph: Find troubleshooting resources for resolving issues with Generative APIs.
dates:
  posted: 2025-03-10
categories:
  - generative-apis
---

<ProductHeader
  productName="Generative APIs troubleshooting"
  productLogo="generativeApi"

>   description="Troubleshoot common issues with Generative APIs on Scaleway. Find solutions to help resolve problems and get started."

Reviewer: ```suggestion
  description="Learn how to fix the most common issues with Generative APIs on Scaleway."
```

/>

## Featured Pages

<Grid>
    <DefaultCard
      title="Generative APIs FAQ"

>       description="General info on Generative APIs"

Reviewer: ```suggestion
      description="General info on Generative APIs."
```

      url="/faq/generative-apis/"
      label="See more"
    />
    <DefaultCard
      title="Fixing common issues with Generative APIs"

>       description="Troubleshoot common issues that you may encounter when using Generative APIs, their causes, and recommended solutions."

Reviewer: ```suggestion
      description="Learn about known issues with Generative APIs, their causes, and recommended solutions."
```

      url="/generative-apis/troubleshooting/fixing-common-issues/"
      label="See more"
    />
    <DefaultCard
      title="Supported models"

>       description="Learn about the different models supppored on Generative APIs"

Reviewer: ```suggestion
      description="Learn about the different models supppored on Generative APIs."
```

      url="/generative-apis/reference-content/supported-models/"
      label="See more"
    />
</Grid>
<Grid>
    <DefaultCard 
      title="Generative APIs rate limits"

>       description="Learn about the rate limits applied to Generative APIs"

Reviewer: ```suggestion
      description="Discover the rate limits applied to Generative APIs models."
```

      url="/generative-apis/reference-content/rate-limits/"
      label="See more"
    />
    <DefaultCard 
      title="Integrating Scaleway Generative APIs with popular AI tools"

>       description="Discover how-to integrate Generative APIs with popular AI tools"

Reviewer: ```suggestion
      description="Explore the AI tools and libraries supported by Scaleway's Generative APIs."
```

      url="/generative-apis/reference-content/integrating-generative-apis-with-popular-tools/"
      label="See more"
    />
    <DefaultCard 
      title="Generative APIs Privacy Policy"

>       description="Inform yourself about the Privcy Policy applied to Generative APIs"

Reviewer: ```suggestion
      description="Inform yourself about the privacy policy applied to Generative APIs."
```
